### PR TITLE
Breakout Room 5

### PR DESCRIPTION
**Pull request by tgoslee**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1 +1,22 @@
 #Mad Libs Generator Project
+loop = 1
+while (loop < 9):
```
Comment (mckay1818) [inline main.py:3]: Loop will currently exit after 8 rounds of Mad Libs - reconfigure this so there are 10 rounds

File: main.py [hunk 2]
```diff
@@ -1 +1,22 @@
...
+    print ("------------------------------------------")
+    print ("Be kind to your",noun,"- footed", plural_noun)
+    print ("For a duck may be somebody's", seond_noun,",")
+    print ("Be kind to your",plural_noun,"in",place)
+    print ("Where the weather is always",adjective,".")
+    print ()
+    print ("You may think that is this the",third_noun,",")
+    print ("Well it is.")
+    print ("------------------------------------------")
+#Loop back to "loop = 1"
```
Comment (mckay1818) [inline main.py:21]: Loop starts at the "while" statement on line 3, not line 2. It could be helpful for you to clarify this in your comment!

File: main.py (hunk 1 above)
Comment (ryjpink) [inline main.py:3]: The game should loop so the user can create 10 stories. The while loop can work without round bracket.
```suggestion
while loop < 11:
```

File: main.py [hunk 3]
```diff
@@ -1 +1,22 @@
...
+    loop = loop + 1
```
Comment (ryjpink) [inline main.py:22]:
```suggestion
    loop += 1
```

File: main.py [hunk 4]
```diff
@@ -1 +1,22 @@
...
+#Displays the story based on the users input
+    print ("------------------------------------------")
+    print ("Be kind to your",noun,"- footed", plural_noun)
+    print ("For a duck may be somebody's", seond_noun,",")
+    print ("Be kind to your",plural_noun,"in",place)
+    print ("Where the weather is always",adjective,".")
+    print ()
+    print ("You may think that is this the",third_noun,",")
+    print ("Well it is.")
+    print ("------------------------------------------")
```
Comment (mckay1818) [inline main.py:20]: To cut down on print statements, you could consider making this a multi-line string! https://www.w3schools.com/python/gloss_python_multi_line_strings.asp

File: main.py [hunk 5]
```diff
@@ -1 +1,22 @@
...
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
+#Displays the story based on the users input
+    print ("------------------------------------------")
+    print ("Be kind to your",noun,"- footed", plural_noun)
+    print ("For a duck may be somebody's", seond_noun,",")
```
Comment (ryjpink) [inline main.py:14]:
```suggestion
    print ("For a duck may be somebody's", second_noun,",")
```

Comment (mckay1818) [inline main.py:14]: Currently, all punctuation after user inputted words is separated from those words with a space (lines 14, 16, 18). You could remove these spaces for better readability

File: main.py (hunk 3 above)
Comment (ryjpink) [reply main.py:22]: Make multiline statements and use [f-string](https://www.geeksforgeeks.org/formatted-string-literals-f-strings-python/) in the print function.

File: main.py (hunk 2 above)
Comment (ryjpink) [inline main.py:21]:
```suggestion
#Loop back to "while loop < 11:"
```

**Review by mckay1818 — COMMENTED** [review]
Great start on the madlibs project! I added some specific suggestions for the main.py file, but I would also recommend adding a tests file and testing for edge cases, # of loops, etc.

**Review by ryjpink — COMMENTED** [review]
Good start on Mad Libs Generator project!!  It would also be good to add a test file to make sure your project working as your expectation.
